util/new_keyboard.sh

```diff
@@ -32,6 +32,8 @@ set_git_username() {
 
 # Copy the template files to the new keyboard directory.
 copy_templates() {
+    mkdir -p "$(dirname $keyboard_dir)"
```

Comment:
```suggestion
    mkdir -p "$keyboard_dir"
```

The `dirname` should not be necessary either.

Reply:
I understood your opinion.
I like copying entire directory, so I kept this `dirname` . it's just my opinion, it doesn't need to keep in qmk repository. I'll fix it.